EMML/8001-9000/EMML8509.xml

```diff
@@ -542,6 +542,7 @@ seventh-row signs (ሖ ኮ አ) have legs of the same length and a little stroke
       </profileDesc>
       <revisionDesc>
          <change who="MKr" when="2023-09-26">Created record</change>
+         <change who="MKr" when="2024-03-20">Corrected work ID in ms_i28</change>
```

Comment:
I think, it is a good idea to mention the msItem, that you changed in your change record! I will do the same in future.